css/_darkvariables.scss

```diff
@@ -2,7 +2,7 @@ $blue-500: #0a84d6; /*light to dark in dark theme*/
 $blue-300: #136fac;
 $blue-100: #1d5a83;
 
-$orange-400: #ffc285;
+$orange-400: #c27c36;
 $orange-500: #FFA726;
```

Comment:
i think we found that the color "brightness" is reversed in the dark theme becouse of how it behaves, so in the dark orange 500 is brighter than orange 400